LLaMA_LoRA.ipynb

```diff
@@ -144,15 +139,17 @@
         "# colab_notebook_name = remove_ipynb_extension(colab_notebook_filename)\n",
         "\n",
         "from google.colab import drive\n",
-        "drive.mount(google_drive_mount_path)\n",
+        "try:\n",
+        "  drive.mount(google_drive_mount_path)\n",
         "\n",
-        "# google_drive_data_directory_relative_path = f\"{google_drive_colab_data_folder}/{colab_notebook_name}\"\n",
-        "google_drive_data_directory_relative_path = google_drive_folder\n",
-        "google_drive_data_directory_path = f\"{google_drive_mount_path}/My Drive/{google_drive_data_directory_relative_path}\"\n",
-        "!mkdir -p \"{google_drive_data_directory_path}\"\n",
-        "!ln -nsf \"{google_drive_data_directory_path}\" ./data\n",
-        "!touch \"data/This folder is used by the Colab notebook \\\"{colab_notebook_filename}\\\".txt\"\n",
-        "!echo \"Data will be stored in Google Drive folder: \\\"{google_drive_data_directory_relative_path}\\\", which is mounted under \\\"{google_drive_data_directory_path}\\\"\"\n"
+        "  google_drive_data_directory_relative_path = google_drive_folder\n",
+        "  google_drive_data_directory_path = f\"{google_drive_mount_path}/My Drive/{google_drive_data_directory_relative_path}\"\n",
+        "  !mkdir -p \"{google_drive_data_directory_path}\"\n",
+        "  !ln -nsf \"{google_drive_data_directory_path}\" ./data\n",
+        "  !touch \"data/This folder is used by the Colab notebook \\\"{colab_notebook_filename}\\\".txt\"\n",
+        "  !echo \"Data will be stored in Google Drive folder: \\\"{google_drive_data_directory_relative_path}\\\", which is mounted under \\\"{google_drive_data_directory_path}\\\"\"\n",
+        "except Exception as e:\n",
+        "  print(\"Drive won't be mounted!\")"
```

Comment:
let's make this part optional, it could scare people away

Reply:
I can't agree with you more.